bodhi/server/bugs.py

```diff
@@ -24,6 +24,7 @@
 
 bugtracker = None
 log = logging.getLogger('bodhi')
+FakeBug = namedtuple('FakeBug', 'bug_id')
```

Comment:
Personally, I would prefer `FakeBug = namedtuple('FakeBug', ['bug_id'])`.

Reply:
Ok, I used `string` because there is only one attribute but I have nothing against using `list` here. Fixed and rebased.